### PR TITLE
US 1583733 Add missing language IDs - 18

### PR DESCRIPTION
**Pull request by v-thpra**
- Used "cpp" for "unmanaged" function signatures.
- Used "cpp" for the IDL file example.
- Used "console" for tlbimp.exe command examples.
- Used "console" for Mefx command examples, even commands combined with output.
- Used "output" for examples of Mefx command output only.

Contributes to #2192

### CODE REVIEW

File: docs/framework/interop/how-to-implement-callback-functions.md [hunk 1]
```diff
@@ -18,9 +18,7 @@ The following procedure and example demonstrate how a managed application, using
   
 1. Look at the signature for the **EnumWindows** function before going further with the implementation. **EnumWindows** has the following signature:  
   
-    ```  
-    BOOL EnumWindows(WNDENUMPROC lpEnumFunc, LPARAM lParam)  
-    ```  
+    `BOOL EnumWindows(WNDENUMPROC lpEnumFunc, LPARAM lParam)`
```
Comment (Youssef1313) [inline docs/framework/interop/how-to-implement-callback-functions.md]: I think adding an identifier here is better.

File: docs/framework/interop/how-to-implement-callback-functions.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ The following procedure and example demonstrate how a managed application, using
   
 1. Look at the signature for the **EnumWindows** function before going further with the implementation. **EnumWindows** has the following signature:  
   
-    ```  
-    BOOL EnumWindows(WNDENUMPROC lpEnumFunc, LPARAM lParam)  
-    ```  
+    ```csharp
```
Comment (rpetrusha) [inline docs/framework/interop/how-to-implement-callback-functions.md]:
```suggestion
    ```cpp
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
I've left one suggestion for you to consider, @v-thpra.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
I'll merge your PR now. Thanks, Tom.

**Pull request merged**